### DISCUSSION TITLE
Mini sites para aprender Japonês

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#2017] [Mostre seu trabalho]
Fala pessoal, ultimamente estou brincando de aprender Japonês e como bom programador, eu resolvi fazer uns projetinhos para me ajudar xD

Até agora eu fiz 2:

- [Learn Japanese - Hiragana and Katakana](https://learnjapanese.vercel.app/): um site que mostra as "letrinhas" japonesas e como pronunciá-las. Usei NextJS (React), Web Speech API e Vercel.

- [Japanese Words - learn a new word everyday](https://japanese-words.vercel.app/) : um site estilo flashcard, que mostra imagem, palavra em hiragana, kanji, significa e também como pronunciar com romanji. Também usei NextJS e Vercel.

Os projetos são opensource, espero que gostem e sejam bem vindos a enviar melhorias! =D

**Comment by aronreisx** [comment]
Cara, que bonito o layout das aplicações. Parabéns!
PS: to pra fazer teu curso de react avançado

**Comment by danweblife** [comment]
Show o visual e layots..parabens !!

https://weblifebrasil.com.br/

**Comment by ghost** [comment]
A paleta de cores ficou muito boa.
